3rd-party/3rd-party.php

```diff
@@ -13,6 +13,7 @@
 	'bitly.php',
 	'buddypress.php',
 	'class.jetpack-amp-support.php',
+	'class.jetpack-amp-vimeo-shortcode.php',
```

Comment:
Instead of being its own file, I wonder if it wouldn't be best to have this part of the existing Vimeo shortcode handler in `modules/shortcodes/vimeo.php`.

This may also allow us to rely on the existing work done there to extract data from Vimeo URLs.

Reply:
Good idea. The commits below apply it, including https://github.com/Automattic/jetpack/pull/13921/commits/c66aff4e3f24950ca57b5ab92598c6e2b66bf20e. But it looks like I still need to fix unit tests.